### PR TITLE
Set up Portfolio: mobile version

### PR DESCRIPTION
**Pull request by nati2323**
- Added "Works" section
- Added "About myself" section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,62 +1,363 @@
...
+                html
+              </p>
+              <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50
+                  ">
+                css
+              </p>
+              <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+                javascript
+              </p>
+            </div>
```
Comment (omar-muhamad) [inline index.html]: - [ ] For all work cards please put the used technologies like (HTML, CSS,...etc.) inside unordered list`<ul>`, `<li>` tags

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @nati2323,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (m05e5) [inline index.html]: - [ ] please kindly follow HTML semantic code by using `ul` for the programming language list and `li` for each programming language.

File: index.html [hunk 2]
```diff
@@ -1,62 +1,363 @@
...
+              <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+                Ruby on rails
+              </p>
+              <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+                css
+              </p>
+              <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">
+                javascript
+              </p>
+            </div>
```
Comment (m05e5) [inline index.html]: - Same thing goes here

File: index.html [hunk 3]
```diff
@@ -1,62 +1,363 @@
...
```
Comment (m05e5) [inline index.html]: - [ ] Same thing goes here

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @nati2323 , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please apply the changes suggested by the previous reviewer on your cards
_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved ✔️
Hi @nati2323 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**